### PR TITLE
Anw 462

### PR DESCRIPTION
**Pull request by lorawoodford**
Add functionality to merge top containers from the Manage Top Containers bulk actions screen.

## Related JIRA Ticket or GitHub Issue

https://archivesspace.atlassian.net/browse/ANW-462

## How Has This Been Tested?

Manually tested (including by testing group); all automated tests passing.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have authority to submit this code.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by lorawoodford** [comment]
@lmcglohon , @cdibella - This now includes @brianzelip 's confirmation modal.  Open for your review.

File: backend/app/model/mixins/relationships.rb [hunk 1]
```diff
@@ -92,58 +92,123 @@ def self.transfer(target, victims)
...
 
       victim_columns.each do |victim_col|
 
-        # Find any relationship where the current column contains a reference to
-        # our victim
-        self.filter(victim_col => victims.map(&:id)).each do |relationship|
+        vics.each do |victim|
+
+          parents = who_participates_with(victim)
+          next if parents.empty?
```
Comment (mark-cooper) [inline backend/app/model/mixins/relationships.rb]: When return value is an array can just use directly: `who_participates_with(victim).each do |parent|`

File: backend/app/model/mixins/relationships.rb [hunk 2]
```diff
@@ -92,58 +92,123 @@ def self.transfer(target, victims)
...
+    if !dups.empty?
+      dups.each {|d| d.delete}
+    end
+  end
 
-          relationship.save
+
+  def self.identify_duplicate_containers(target, relationship, target_col, dups)
+    target_relationships = find_by_participant(target)
+    target_relationships.each do |target_relationship|
```
Comment (mark-cooper) [inline backend/app/model/mixins/relationships.rb]: Same as above, can be reduced to: `find_by_participant(target).each do |target_relationship|`

**Comment by lorawoodford** [comment]
@mark-cooper - Added a test that replicates the situation that was causing duplicate linked subjects to remain post-merge prior to b933311546f4e2092f86045cba3a87ef43ba051b

**Comment by mark-cooper** [comment]
@lorawoodford 👍

**Pull request merged**